cmd/collector/collector.go

```diff
@@ -130,7 +130,7 @@ func run() error {
 		return fmt.Errorf("input given ipfix.transport flag is not supported or valid")
 	}
 	// Initialize collecting process
-	cp, err := collector.InitCollectingProcess(netAddr, 65535, 0)
+	cp, err := collector.InitCollectingProcess(netAddr, 65535, 0, false, nil, nil)
```

Comment:
For public methods, we can consider creating a struct for all the input parameters.
This can avoid changing the code on the caller side whenever we want to add new parameters in the future.
I was asked by WCP team to do something like that when they consume a library maintained by our team. Otherwise every time we upgrade our library, that could break their existing code.

Reply:
Thanks it will also be clearer for users to pass parameters. Updated for exporter and intermediate process too.

Reply:
@shihhaoli It is a good suggestion to make consumption easier. 
However, it is better to avoid company-specific internal links in the repo.

Reply:
Yes, I made the same mistake while developing openstack long time ago (referring bugzilla link for bug fix in the commit message) :-)
Any suggestion how to do this in the future? Send an email instead?

Reply:
I felt if there are non-vmware folks following the conversation, they may not get complete context with the links that are not accessible. Yes offline mode through email or slack sounds good to me.

Reply:
OK, then those non-vmware people will completely hide from our conversations, but at least they will not complain inaccessible links : -)

Reply:
Or keeping the context to an accessible project will also work :)